### PR TITLE
APIGOV-29233 Remove Platform logout confirmation dialog when using axway cli auth logout

### PR DESCRIPTION
**Pull request by kasuvy**
Remove Platform logout confirmation dialog when using axway cli auth logout

### CODE REVIEW

File: packages/amplify-sdk/src/endpoints.js [hunk 1]
```diff
@@ -21,10 +21,12 @@ export default function getEndpoints({ baseUrl, realm } = {}) {
 	// strip the trailing slashes
 	baseUrl = baseUrl.replace(/\/$/, '');
 
+	platformUrl = platformUrl.replace(/\/$/, '');
+
 	return {
 		auth:      `${baseUrl}/auth/realms/${realm}/protocol/openid-connect/auth`,
 		certs:     `${baseUrl}/auth/realms/${realm}/protocol/openid-connect/certs`,
-		logout:    `${baseUrl}/auth/realms/${realm}/protocol/openid-connect/logout`,
+		logout:    `${platformUrl}/auth/signout`,
```
Comment (pkclark) [inline packages/amplify-sdk/src/endpoints.js:29]: -1 on adding non-keycloak ref URLs herein.

File: packages/amplify-sdk/src/amplify-sdk.js [hunk 2]
```diff
@@ -405,9 +405,9 @@ export default class AmplifySDK {
 					if (account.isPlatform && !account.isPlatformTooling) {
 						// note: there should only be 1 platform account in the accounts list
 						const { platformUrl } = environments.resolve(account.auth.env);
-						const { logout } = getEndpoints({ baseUrl: account.auth.baseUrl, realm: account.auth.realm });
-						const redirect = `${logout}?redirect_uri=${platformUrl}/signed.out?msg=signout`;
-						const url = `${platformUrl}/api/v1/auth/logout?redirect=${encodeURIComponent(redirect)}`;
+						const { logout } = getEndpoints({ baseUrl: account.auth.baseUrl, platformUrl: platformUrl, realm: account.auth.realm });
```
Comment (pkclark) [inline packages/amplify-sdk/src/amplify-sdk.js:408]: Given that we're constructing the redirect path off `platformUrl` below, perhaps we just do the same and remove `logout` from that endpoint file.

**Comment by pkclark** [comment]
@kasuvy updates look good. There's some linting founds that need addressed.

File: packages/axway-cli-pm/src/pm.js [hunk 3]
```diff
@@ -357,6 +358,7 @@ export async function search({ keyword, limit, type } = {}) {
 	await Promise.all(packages.map(({ name, version }) => {
 		return plimit(async () => {
 			try {
+				console.log(name);
```
Comment (vivekschauhan) [inline packages/axway-cli-pm/src/pm.js:361]: This can be removed

File: packages/axway-cli-pm/src/pm.js [hunk 4]
```diff
@@ -382,6 +383,7 @@ export async function view(pkgName, { requestOpts = createRequestOptions(), type
 	if (!pkgName || typeof pkgName !== 'string') {
 		throw new TypeError('Expected package name to be a non-empty string');
 	}
+	console.log(pkgName);
```
Comment (vivekschauhan) [inline packages/axway-cli-pm/src/pm.js:386]: This can be removed

**Pull request merged**